components/roadmap-grid/RoadmapTabbedView.tsx

```diff
@@ -67,15 +66,17 @@ export function RoadmapTabbedView({
     }
 
     return (
-      <Tab
+      <Skeleton isLoaded={!globalLoadingState.get()}>
```

Comment:
Shouldn't the tabs always be visible? but the tab panels render skeletons when content isn't ready?

Reply:
any UI component that can be rendered, should be rendered, as soon as possible. That's the whole point of the speed improvements isn't it?

The idea of skeleton loading is to reduce Cumulative Layout Shift([CLS](https://web.dev/cls/)) while communicating that data is loading. If the primary navigational UI components are always in the same spot without needing data to be rendered, then there is less CLS.